src/visu_config.xsd

```diff
@@ -924,8 +924,13 @@
       <xsd:element name="label" type="label" minOccurs="0" maxOccurs="1"/>
       <xsd:element name="address" type="address" minOccurs="0" maxOccurs="unbounded"/>
     </xsd:sequence>
-    <xsd:attribute ref="value" use="required" />
-    <xsd:attribute name="shortvalue" type="xsd:string" use="optional" />
+    <xsd:attribute ref="value" use="required"/>
+    <xsd:attribute name="shortvalue" type="xsd:string" use="optional">
+      <xsd:annotation>
+      	  <xsd:documentation xml:lang="en">Defines the value which is sent on short push of the button</xsd:documentation>
+      	  <xsd:documentation xml:lang="de">Legt fest welcher Wert bei einem kurzen Tastendruck gesendet wird.</xsd:documentation>
```

Comment:
Maybe a little bit "sperrig" but in lack of a better alternative I think it's acceptable. Maybe someone else has a better idea.